### ISSUE TITLE
code tag in footnotes results in 'undefined' error

### ISSUE THREAD

**Issue by nimishgautam** [TryGhost/Ghost#6840]
### Issue Summary

If you use either backticks to represent code, or the CODE tag inside a footnote, the contents of the tag are replaced with the word "undefined"
### Steps to Reproduce
1. Type the following into a new blog post :

```
This is normal text.[^1]

`this` is a single line of code [^2]

[^1]: this is still ok but `this` causes an error
[^2]: <code>this</code> will also go crazy
```
1. In the 'preview' pane, the contents of the code blocks will be replaced with the word 'undefined', same if you publish
### Technical details:
- Ghost Version: 0.7.8 (I checked and this also is an issue on the ghost pro version being hosted by ghost.org)
- Node Version: v0.10.25
- Browser/OS: Chrome/OSX
- Database: sqlite

**Comment by ErisDS** [comment]
@nimishgautam Thanks for the detailed bug report 👍  I'll take a look at getting a fix in as soon as possible.

**Comment by vihanb** [comment]
I've recently updated to ghost version v0.11.10 and I still have this issue 
<img width="649" alt="screen shot 2017-07-11 at 7 59 16 pm" src="https://user-images.githubusercontent.com/15314511/28099800-71aaf0a8-6673-11e7-977b-3b5bb0e4d94e.png">

This also appears when I click "Preview' and it opens up the preview in a separate page.

**Comment by kevinansfield** [comment]
@vihanb this issue is fixed in 1.0, it won't be addressed in the 0.11.x versions as the fix requires some potentially breaking changes
